packages/dev-utils/src/lib/geo-location.ts

```diff
@@ -10,3 +12,105 @@ export const mockLocation: Geolocation = {
   latitude: 0,
   timezone: 'UTC',
 }
+
+const API_URL = 'https://netlifind.netlify.app'
+const STATE_GEO_PROPERTY = 'geolocation'
+// 24 hours
+const CACHE_TTL = 8.64e7
+
+// 10 seconds
+const REQUEST_TIMEOUT = 1e4
+
+/**
+ * Returns geolocation data from a remote API, the local cache, or a mock location, depending on the
+ * specified options.
+ */
+export const getGeoLocation = async ({
+  geoCountry,
+  enabled = true,
+  cache = true,
+  offline = false,
+  state,
+}: {
+  enabled?: boolean
+  cache?: boolean
+  geoCountry?: string | undefined
+  offline?: boolean | undefined
+  state: LocalState
+}): Promise<Geolocation> => {
```

Comment:
this has become redundant. we only need to support `enabled` and `cache` now, so you can remove `offline`.